msteams-platform/toolkit/Teams-Toolkit-CLI.md

```diff
@@ -12,27 +12,20 @@ zone_pivot_groups: toolkit-cli
 # Teams Toolkit command line interface
 
 > [!IMPORTANT]
->
-> * Teams Toolkit CLI v3 is available in beta version.
-> * We recommend that you use Teams Toolkit CLI v3 to build your Teams app. TeamsFx CLI v1 and TeamsFx CLI v2 are soon deprecated.
+> We recommend that you use Teams Toolkit CLI v3 to build your Teams app. TeamsFx CLI v1 and TeamsFx CLI v2 are soon deprecated.
 
```

Comment:
Can we mention that v1 is no longer supported, v2 will be end of support by March 2025 (One year after v3 officially released)

Reply:
Added note that TeamsFx CLI v1 isn't supported and TeamsFx CLI v2 will be depreated by March 2025..